src/test/java/tw/commonground/backend/service/fact/FactServiceTest.java

```diff
@@ -0,0 +1,189 @@
+package tw.commonground.backend.service.fact;
+
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.*;
+import org.mockito.junit.jupiter.MockitoExtension;
+import tw.commonground.backend.service.fact.entity.FactRepository;
+import tw.commonground.backend.service.reference.ReferenceEntity;
+import tw.commonground.backend.service.reference.ReferenceRepository;
+
+import java.io.IOException;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+@SuppressWarnings("MethodName")
+@ExtendWith(MockitoExtension.class)
+public class FactServiceTest {
```

Comment:
JUnit5 test classes and methods should have default package visibility

ref: https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods (see Class and method visibility)